sklearn/decomposition/_fastica.py

```diff
@@ -471,24 +470,24 @@ def g(x, fun_args):
             )
 
         if self.whiten:
-            # Centering the columns (ie the variables)
-            X_mean = X.mean(axis=-1)
-            X -= X_mean[:, np.newaxis]
+            # Centering the features of X
+            XT_mean = XT.mean(axis=-1)
+            XT -= XT_mean[:, np.newaxis]
```

Comment:
I think we could keep the `X_mean` name, as this variable stores the feature-wise means of X, that is `X.mean(axis=0)` and `axis=0` is the intuitive default (I believe).

Reply:
Make sense! I've changed it changed it back to X_mean